presentation/src/main/java/com/example/presentation/navigation/MashUpNavHost.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.presentation.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.navigation
+import androidx.navigation.navArgument
+import com.example.presentation.ui.detail.DetailRoute
+import com.example.presentation.ui.home.HomeRoute
+import com.example.presentation.ui.home.HomeViewModel
+
+const val FOLLOWER_INDEX = "index"
+
+@Composable
+fun MashUpNavHost(
+    modifier: Modifier = Modifier,
+    navController: NavHostController,
```

Comment:
```suggestion
    navController: NavHostController
    modifier: Modifier = Modifier,
```

kotlin 매개변수 순서에 따르면 디폴트 값이 있는 매개변수가 아래로 내려가면 좋을 것 같아요 :)

Reply:
@jaeryo2357 저도 몰랐었는뎅 감사합니다~ 붐업 드립니다 👍